docs/book/src/SUMMARY.md

```diff
@@ -8,6 +8,16 @@
     - [Certificate Management](./tasks/certs/index.md)
         - [Using Custom Certificates](./tasks/certs/using-custom-certificates.md)
         - [Generating a Kubeconfig](./tasks/certs/generate-kubeconfig.md)
+- [clusterctl](./clusterctl/intro.md)
+    - [init](./clusterctl/init.md)
+    - [move](./clusterctl/move.md)
+    - [upgrade](./clusterctl/upgrade.md)
+    - [config cluster](clusterctl/config-cluster.md)
+    - [adopt](clusterctl/adopt.md)
+    - [delete](clusterctl/delete.md)
```

Comment:
Let's put these under`clusterctl Commands`

Reply:
I assume this should be a new subtree in the summary